client/web/src/components/widgets/inputs/NumberInput.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-	<div class="number-input">
-		<button class="arrow left" @click="onIncrement(-1)"></button>
-		<button class="arrow right" @click="onIncrement(1)"></button>
-		<input type="text" spellcheck="false" v-model="text" @change="updateText($event.target.value)" /> />
+	<div class="number-input" :class="{ disabled }">
```

Comment:
You can just do `class="static"`, right?

Reply:
I don't understand, could you please elaborate on this?

Reply:
Binding `:class` to a constant list is redundant, as I think you can just use `class="disabled"`

Reply:
`{ disabled }` is the ES6 shorthand for `{ "disabled": disabled }` which is conditionally setting the `disabled` class on the component based on `this.disabled`.

Reply:
Ah got it, all good then